### PR TITLE
Update feature-gates.md

### PR DESCRIPTION
**Pull request by stewart-yu**
add `DynamicProvisioningScheduling` feature-gate docs

Allow edits from maintainers checkbox

### CODE REVIEW

**Comment by stewart-yu** [comment]
/hold 
will add some content about it in others directory

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ different Kubernetes components.
 | `DevicePlugins` | `true` | Beta | 1.10 | |
 | `DynamicKubeletConfig` | `false` | Alpha | 1.4 | 1.10 |
 | `DynamicKubeletConfig` | `true` | Beta | 1.11 | |
+| `DynamicProvisioningScheduling` | `false` | Alpha | 1.11 | 1.11 |
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md]: Is the feature introduced in 1.11 and it is removed in 1.12?

Comment (stewart-yu) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: add in v1.11, but seems not finish completed, https://github.com/kubernetes/kubernetes/pull/63232/commits/95b530366aeea7c379c50424f58bcd2f27bed921

Comment (tengqm) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: Then you should remove `1.11` from the last column

**Comment by neolit123** [comment]
@stewart-yu any update on the change requested by @tengqm ?

**Comment by stewart-yu** [comment]
Sorry for delay.
I want to add some contents about `DynamicProvisioningScheduling`, but busy in `core repo`, may be as `follow-up`
This PR want to merge first
/hold cancel

**Pull request merged**